Tests/StreamChatUITests/SnapshotTests/ChatThread/ChatThreadVC_Tests.swift

```diff
@@ -78,6 +78,34 @@ final class ChatThreadVC_Tests: XCTestCase {
         )
     }
 
+    func test_whenShouldMessagesStartAtTheTopEnabled() {
```

Comment:
```suggestion
    func test_whenShouldMessagesStartAtTheTopIsTrue() {
```
As this is a boolean value, true/false is a more accurate match.